lib/steps/setupRedirects.js

```diff
@@ -44,7 +44,16 @@ const setupRedirects = () => {
     // One route may map to multiple Netlify routes: e.g., catch-all pages
     // require two Netlify routes in the _redirects file
     getNetlifyRoutes(route).map((netlifyRoute) => {
-      const redirect = `${netlifyRoute}  ${nextRedirect.target}  200`;
+      const { conditions, force, statusCode, target } = nextRedirect;
```

Comment:
```suggestion
      const { conditions = [], force = false, statusCode = 200, target } = nextRedirect;
```

Is it possible to set defaults like this?

Reply:
ye good call :)